moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -56,59 +57,51 @@ const useColumns = ({ deleteInProgress, onDeleteActivity, setEditActivity }) =>
       {
         headerName: "ID",
         field: "reference_id",
-        minWidth: 125,
-        flex: 1,
+        width: 125,
         defaultVisible: true,
       },
       {
         headerName: "Workgroup/Contractor",
         field: "workgroup_contractor",
-        minWidth: 175,
-        flex: 1,
+        width: 175,
         defaultVisible: true,
       },
       {
         headerName: "Contract #",
         field: "contract_number",
-        minWidth: 150,
-        flex: 1,
+        width: 150,
         defaultVisible: true,
       },
       {
         headerName: "Description",
         field: "description",
-        minWidth: 200,
-        flex: 1,
+        width: 200,
         defaultVisible: true,
       },
       {
         headerName: "Work Assignment",
         field: "work_assignment_id",
-        minWidth: 150,
-        flex: 1,
+        width: 150,
         defaultVisible: true,
       },
       {
         headerName: "Task Order(s)",
         field: "task_orders",
         defaultVisible: true,
-        minWidth: 150,
-        flex: 1,
-        valueGetter: ({ row }) =>
-          row.task_orders?.map((tk) => tk.task_order).join(", "),
+        width: 200,
+        valueGetter: (field) => field?.map((tk) => tk.display_name).join(", "),
         renderCell: ({ row }) => (
           <div>
             {row.task_orders?.map((tk) => (
-              <div key={tk.task_order}>{tk.task_order}</div>
+              <div key={tk.task_order}>{tk.display_name}</div>
```

Comment:
@amenity i switched this to be the display name, I know you already reviewed the PR, but just letting you know

Reply:
Thank you!